scripts/build-vcpkg.sh

```diff
@@ -7,6 +7,7 @@ cd /usr/src/qfield || exit
 CMAKE_BUILD_DIR=/usr/src/qfield/build-${triplet}
 
 export ANDROID_NDK_HOME=${ANDROID_NDK_ROOT}
+export Qt5_Dir=/home/devel/5.14.2/android/
 export Qt5_DIR=/home/devel/5.14.2/android/
```

Comment:
Can we drop this line instead?

Reply:
@m-kuhn , sure, if only one is needed :)

Reply:
We can't, just tested.